src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,31 @@
+<div class='container-dashboard'>
```

Comment:
In un componenete che si chiama `dashboard.component` non mi aspetterei di trovare direttamente cose che fanno riferimento ad animali. Questo andrebbe sistemato, e' confusionario.